commons/connector/altoida/upload_altoida_action.avsc

```diff
@@ -0,0 +1,12 @@
+{
+  "namespace": "org.radarcns.connector.altoida.AltoidaAction",
+  "type": "record",
+  "name": "AltoidaAction",
+  "doc": "Logs of actions taken.",
+  "fields": [
+    {"name": "time", "type": "double", "doc": "Timestamp in unix time acquired locally from the device."},
+    {"name": "timeReceived", "type": "double", "doc": "Timestamp in unix time received from Altoida API."},
+    {"name": "logActions", "type": "string", "doc": "Log of actions taken during test."},
```

Comment:
Is it log of multiple actions or a single action?

Reply:
This is a single action. So I changed al names to the singlular form